docs/dev-guide/integration/rest-api/content/associate.rst

```diff
@@ -40,7 +40,6 @@ with an importer that supports the type of units being copied.
       }
     },
    'override_config': {
-     'resolve_dependencies: true,
```

Comment:
Does this mean we had documented the code we are removing? If so, how can we be sure that removing this API is OK without waiting until 3.0?